src/models/Account.js

```diff
@@ -8,17 +8,15 @@ const AccountSchema = new Schema({
         trim: true,
         lowercase: true,
         unique: true,
-        required: "Email address is required",
+        required: true,
     },
     password: { type: String, required: true },
     isAdmin: {
         type: Boolean,
-        required: function() {
-            return !this.companyRef;
-        },
+        default: false,
         validate: {
             validator: function(isAdmin) {
-                return isAdmin && !this.company;
+                return isAdmin !== !!this.company;
             },
```

Comment:
Can we swap to arrow function and make this a one-liner?

```js
validator: isAdmin => isAdmin !== !!this.company;
```

Reply:
Arrow functions do not capture this, hence why the validators use the older `function` syntax :sweat_smile:

Reply:
Correct.. you would have to `.bind(this)`... nevermind